package/MDAnalysis/lib/_distopia.py

```diff
@@ -36,17 +36,15 @@
 else:
     HAS_DISTOPIA = True
 
-from .c_distances import (
-    calc_bond_distance_triclinic as _calc_bond_distance_triclinic_serial,
-)
+
 import warnings
 import numpy as np
 
 
 def calc_bond_distance_ortho(
     coords1, coords2: np.ndarray, box: np.ndarray, results: np.ndarray
 ) -> None:
-    distopia.calc_bonds_ortho_float(
+    distopia.calc_bonds_ortho(
```

Comment:
Codecov says not covered by tests? Is codecov wrong? If it's right then need to make sure that the new code is actually called in the tests.

Reply:
Codecov is wrong, but I am not exactly sure why its not picking up the conditional.

Reply:
Codecov now sees them as tested. ✅